sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -453,12 +456,16 @@ def read_porttab_mappings(self, porttabfile):
                 # so we use the port's position in the file (zero-based) as bcm_port
                 portname = line.split()[0]
 
+                # Ignore if this is an internal backplane interface
+                if portname.startswith(daemon_base.get_internal_interface_prefix()):
+                    continue
```

Comment:
Is json format file depracated. I don't see similar check for json format files.

Reply:
We do support only the port_config.ini files for multi_asic. The json format files was recently introduced. So I was currently adding the logic to handle multiple port_config.ini files alone.

Reply:
All port_config.ini/platform.json parsing should eventually be moved to sonic-py-common so that it is only done in one place.

Reply:
@jleveque I could move the json/ini file parsing from here --> to  portconfig.py. But what I find is that I will have to **write this logic as new API's** in  portconfig.py -- as the port_config.ini/platform.json parsing logic implemented here in sfpbase module is different to that is already present @ https://github.com/Azure/sonic-buildimage/blob/da69d57a8dffa37e285d7b93671d055ba193c61d/src/sonic-config-engine/portconfig.py#L142.

We could have separate parsing API's for now, and later on unify them in portconfig.py as a single parsing API ?

Reply:
In that case, It's probably best to leave it here for now and do the move and unification at the same time, and maybe also move it all to sonic-py-common at the same time, as well.